PacketTunnelProvider/NetworkProtection/NetworkProtectionPacketTunnelProvider.swift

```diff
@@ -322,6 +328,20 @@ final class NetworkProtectionPacketTunnelProvider: PacketTunnelProvider {
 
         let settings = VPNSettings(defaults: .networkProtectionGroupDefaults)
 
+        Configuration.setURLProvider(VPNAgentConfigurationURLProvider())
+        configurationManager = ConfigurationManager(store: configurationStore)
+        configurationManager.start()
+        let privacyConfigurationManager = VPNPrivacyConfigurationManager.shared
+        // Load cached config (if any)
+        privacyConfigurationManager.reload(etag: configurationStore.loadEtag(for: .privacyConfiguration), data: configurationStore.loadData(for: .privacyConfiguration))
+
+        configuationSubscription = privacyConfigurationManager.updatesPublisher
+            .sink {
+                if privacyConfigurationManager.privacyConfig.isSubfeatureEnabled(BackgroundAgentPixelTestSubfeature.pixelTest) {
+                    DailyPixel.fire(pixel: .networkProtectionConfigurationPixelTest)
+                }
```

Comment:
I think it's possible we'll still miss calls to this pixel - this publisher is only called when we receive updates, so if no Privacy Config update is published on a given day then we'll get no events coming through.

We could probably add the `BackgroundAgentPixelTestSubfeature.pixelTest` check and pixel call somewhere that we _know_ will get called regularly, like `handleConnectionStatusChange`. That said, we only really care about getting these calls on days where we're changing the rollout param and at that point we _will_ get this publisher called, so I'd be fine with it staying as-is.

Reply:
Struggling a bit to follow what `pixelTest` is.